refactor: ♻️ match our 'tagline' to what we actually do (#430)
seedcase-project/seedcase-project: index.qmd

![](_extensions/seedcase-project/seedcase-theme/logos/seedcase/font.svg){fig-alt="Seedcase
font." width="250px"}

> # Growing FAIR and tidy data

Reviewer: And why not "An open ecosystem for" anymore?

Author: Related to you mentioning what our "core work" is in one of the decision PRs. Our core work is building FAIR and tidy data, with the side effect of building tools for that. Seedcase isn't just about the tools, it's us doing the work too. On top of docs, templates, (future) workshops, etc.

::::::: hero-text
## What we do

- We collaborate with research groups to engineer and build up data as FAIR

>   (Findable, Accessible, Interoperable, and Reusable) and tidy packages

Reviewer: ```suggestion
  (Findable, Accessible, Interoperable, and Reusable) and tidy data packages
```

- We develop an ecosystem open-source tools that automate and streamline how we

>   and others can build these data packages.

Reviewer: ```suggestion
- We develop an ecosystem of open-source tools that automate and streamline the
  creation and maintenance of these data packages.
```

Reviewer: Could also be simplified to be:
```suggestion
- We develop open-source tools that automate and streamline how we
  and others can build these data packages.
```

- We contribute towards upskilling and empowering researchers and data owners by

>   mentoring and training them and by writing guides and tutorials.

Reviewer: ```suggestion
- We upskill and empower researchers and data owners through
  mentoring, training, and the development of guides and tutorials.
```
